spaghetto/src/Spago/Config.purs

```diff
@@ -29,54 +30,55 @@ module Spago.Config
   , readWorkspace
   , sourceGlob
   , toManifest
+  , gitPackageCodec
   ) where
 
 import Spago.Prelude
 
 import Affjax.Node as Http
 import Affjax.ResponseFormat as Response
 import Affjax.StatusCode (StatusCode(..))
-import Data.Argonaut.Core as Core
 import Data.Array as Array
-import Data.Bifunctor (lmap)
+import Data.Codec.Argonaut as CA
+import Data.Codec.Argonaut as Codec
```

Comment:
You could remove this and only have the `CA` import.

Reply:
Good catch!